modules/ztp-creating-a-validator-inform-policy.adoc

```diff
@@ -0,0 +1,94 @@
+//
+// *scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-creating-a-validator-inform-policy_{context}"]
+= Creating a validator inform policy
+
+Use the following procedure to create a validator inform policy that provides an indication of
+when the zero touch provisioning (ZTP) installation and configuration of the deployed cluster is complete. This policy
+can be used for deployments of single node clusters, three-node clusters, and standard clusters.
+
+.Procedure
+
+. Create a standalone `PolicyGenTemplate` custom resource (CR) that contains the source file
+`validatorCRs/informDuValidator.yaml`.
+You only need one standalone `PolicyGenTemplate` CR for each cluster type.
++
+* Single node clusters:
++
+[source,yaml]
+----
+group-du-sno-validator-ranGen.yaml
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  name: "group-du-sno-validator" <1>
+  namespace: "ztp-group" <2>
+spec:
+  bindingRules:
+    group-du-sno: "" <3>
+  bindingExcludedRules:
+    ztp-done: "" <4>
+  mcp: "master" <5>
+  sourceFiles:
+    - fileName: validatorCRs/informDuValidator.yaml
+      remediationAction: inform <6>
+      policyName: "du-policy" <7>
+----
+
+* Three-node clusters:
++
+[source,yaml]
+----
+group-du-3node-validator-ranGen.yaml
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  name: "group-du-3node-validator" <1>
+  namespace: "ztp-group" <2>
+spec:
+  bindingRules:
+    group-du-3node: "" <3>
+  bindingExcludedRules:
+    ztp-done: "" <4>
+  mcp: "master" <5>
+  sourceFiles:
+    - fileName: validatorCRs/informDuValidator.yaml
+      remediationAction: inform <6>
+      policyName: "du-policy" <7>
+----
+
+* Standard clusters:
++
+[source,yaml]
+----
+group-du-standard-validator-ranGen.yaml
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  name: "group-du-standard-validator" <1>
+  namespace: "ztp-group" <2>
+spec:
+  bindingRules:
+    group-du-standard: "" <3>
+  bindingExcludedRules:
+    ztp-done: "" <4>
+  mcp: "worker" <5>
+  sourceFiles:
+    - fileName: validatorCRs/informDuValidator.yaml
+      remediationAction: inform <6>
+      policyName: "du-policy" <7>
+----
+<1> The name of `PolicyGenTemplates` object. This name is also used as part of the names
+for the `placementBinding`, `placementRule` and `policy` that are created in the requested `namespace`.
+<2> This value should match the `namespace` used in the group `PolicyGenTemplates`.
+<3> The `group-du-*` label defined in `bindingRules` must exist in the `SiteConfig` files.
+<4> The label defined in `bindingExcludedRules` must be`ztp-done: “”`. The `ztp-done` label is used in coordination with the Topology Aware Lifecycle Operator(TALO).
+<5> `mcp` defines the MachineConfigPool that is used in the source file `validatorCRs/informDuValidator.yaml`. It should be `master` for single node and three-node cluster deployments and `worker` for standard cluster deployments.
```

Comment:
Either:

"the machine config pool"

or

"the `MachineConfigPool` object"

Reply:
"the MachineConfigPool object"